src/fizzBuzz.js

```diff
@@ -0,0 +1,20 @@
+export default function fizzBuzz() {
+  var numbers = []
+  for (var number = 1; number <= 100; number++) {
+
+    if (number % 3 == 0 && number % 5 == 0){
```

Comment:
Using === is usually safer. In this case, we're always dealing with numbers, so it won't cause any issues